### ISSUE TITLE
fix site deploy

### ISSUE THREAD

**Issue by HassanBahati** [OSCA-Kampala-Chapter/evently#9]
![Screenshot from 2023-03-13 16-24-12](https://user-images.githubusercontent.com/65954740/224714746-d7a8d7fc-3d5b-49fb-b9d4-7d021aa584fd.png)

**Comment by mosespace** [comment]
@HassanBahati i guess you can do this right?

**Comment by mosespace** [comment]
Okay,

**Comment by HassanBahati** [comment]
i actually managed to set up the domain at `evently.react-ui.org`.
Though the deploy still fails due to 
https://answers.netlify.com/t/internal-error-during-edge-functions-bundling/87475/5
https://github.com/netlify/next-runtime/issues/1940

trying to find a work around

**Comment by mosespace** [comment]
Actually I have tried running it locally and getting [Unhandled Runtime Error!](https://nextjs.org/docs/messages/react-hydration-error) I think it might be the cause of the bug!😏 What do you think about it @HassanBahati ?
> 
![image](https://user-images.githubusercontent.com/111237257/226113517-678688a3-b0f8-4820-bbb7-606bed92524a.png)

**Comment by HassanBahati** [comment]
oh yess @mosespace true there is a hydration error...but #11 solves it.

let me merge #11 and that error will be fixed....though the deploy seems to still be an issue

**Comment by HassanBahati** [comment]
please pull from the main and that error will be fixed

**Comment by mosespace** [comment]
Hi @HassanBahati , I've pulled the main branch but I think GH-11 did not resolve this because to my surprise I think am still getting the same error code! 
## What could the problem be? 

![image](https://user-images.githubusercontent.com/111237257/226170394-c6739b27-c7c8-4140-b44c-fe074b9744a8.png)

**Comment by HassanBahati** [comment]
@mosespace please try to delete the nodules folder and reinstall/update your dependencies with `pnpm install`

Looking forward to know knowing the result

**Comment by HassanBahati** [comment]
@mosespace your fork seems to be `17 commits` behind the `main` here..please first sync your fork with their `main` at this repository. This should happen by clicking the `Sync Fork` button at your fork

**Comment by mosespace** [comment]
@HassanBahati  Oooh, I had forgotten to `Sync Fork` 🤝
![image](https://user-images.githubusercontent.com/111237257/226195321-c9858260-74af-4f62-95a0-ab91d911c98b.png)

**Comment by mosespace** [comment]
@HassanBahati  I think am yet to finding a **`solution`** to this problem!

**Comment by HassanBahati** [comment]
hello @mosespace sorry i have really caught up past few days...thank you for the resource shared..please let me look into by end of today

**Comment by mosespace** [comment]
Hi, @HassanBahati . Hope your good now, kindly you can take your time to look in it ✨

**Comment by HassanBahati** [comment]
yes i am good...looking it today....we should be having an mvp for this project...

**Comment by HassanBahati** [comment]
@mosespace maybe to keep on track with project goals....this project is intended to enable developers know what events are happening and when they will be happening.

so we are building a catalog for events.....what you may refer to as a `developers events calender`

someone people know about what events are happening though not every one does,,,,yet so many would love to attend...we are trying to solve that for developers, techies, or anyone with interest in tech

**Comment by mosespace** [comment]
@HassanBahati Great to hear about the project goals! It sounds like we're building a platform that will provide a valuable resource for developers and tech enthusiasts who want to stay up to date with upcoming events.

To keep on track with the project goals, it might be helpful to establish a clear timeline for development and set specific milestones to work towards. Additionally, how do you consider reaching out to potential users or stakeholders to gather feedback and ensure that the platform is meeting their needs.

Best of luck to the project!
